paper/paper.md

```diff
@@ -0,0 +1,337 @@
+instructors can gauge learner understanding of the material.
+
+We also incorporate a few additional key components into the curriculum:
+
+1. Each lesson builds off of others, with the goal of creating a final report
+that can be shared with others.
+1. We structured the curriculum such that it could be taught through an
+in-person or virtual workshop. Virtual workshops are sometimes necessary, as during the COVID-19 pandemic, but are also useful to allow people from a variety of geographic locations to instruct and attend.
+1. We require learners to install all software before the workshop (as The
+Carpentries also requires), and ask them to run an example script that tests
+whether everything is installed correctly.
+Learners must send screenshots of the script output to the workshop lead in
+advance in order to attend. This ensures that any installation issues can be
+addressed before the workshop begins.
+1. An extensive small group practice module towards the end of the workshop
+allows learners to more independently practice the skills they have learned.
+1. The workshop concludes with a recap of what was covered and resources
+available for learners to continue learning and getting help as their skills
+develop.
+
+## Pilot Workshop
+<!-- experience of use -->
+
+We piloted our curriculum during a virtual two-day Software Carpentry workshop.
+In line with The Carpentries recommendations
+[@the_carpentries_carpentries_2018], we had four instructors and six helpers at
+the workshop to assist with learner questions and technical issues.
+We also used the "sticky note" system for formative assessment, where learners
+indicate their progress on exercises and request for help by using different
+colored sticky notes [@becker_responding_2016; @the_carpentries_live_2018].
+As the workshop was virtual, we used Zoom reaction icons as virtual sticky
+notes, with the red X reaction to ask for help and the green checkmark to
+indicate that an exercise was successfully completed.
+We had thirty-nine learners of various skill levels from several different
+countries, all of whom provided very positive reviews of the workshop.
+By the end of the workshop, learners on average felt more confident writing programs, using programming to work with data, overcoming problems while programming, and searching for answers to technical questions online (n = 14 survey respondents; pre- and post-workshop surveys).
+All attendees who filled out the post-workshop survey (n = 19) would  recommend the workshop to others.
+
+### Virtual Workshop Reflection
+
+We credit the success of our first virtual workshop in large part due to the
+curriculum structure and content, as well as the instructors and helpers
+involved.
+However, we also believe that the following helped make the workshop as
+smooth as possible:
+
+1. We suggested that learners have Zoom and RStudio (or the Unix shell) open side-by-side on their computer to minimize toggling between different panels.
```

Comment:
Should we reference that Carpentries blog post here?

Reply:
yes, will do!